### ISSUE TITLE
Save States don't apply patches to a game

### ISSUE THREAD

**Issue by MasterChief747** [RPCS3/rpcs3#15895]
### Quick summary

When playing a save state, the patch that fixes issues doesn't apply to the save state. The patch works on the game without using a save state. Is there a way to get the patch working with the save state? Thank you!

### Attach a log file

n/a

**Comment by Nekotekina** [comment]
Sounds very unrealistic to be honest, unless the executable code was not changed. If it was changed, maybe some livepatch-alike techniques might work, but I can't imagine the amount of work required.

**Comment by MasterChief747** [comment]
In this article not sure if they found a solution to it. I couldn't understand what the link they referred to wanted you to do to fix it. Do you understand it? Thank you and have a good day!

https://github.com/RPCS3/rpcs3/issues/12463

**Comment by AniLeo** [comment]
@MasterChief747 stop flooding closed tickets with the same thing, otherwise you will be removed for spam. This ticket already exists to discuss the issue.

**Comment by MasterChief747** [comment]
I am sorry AniLeo. I did not mean to do that. I was just trying to see if different people knew about it. One ticket seemed to say it was solved so I was asking them how they fixed the issue. Did you happen to know how to fix it? Thank you.

**Comment by MasterChief747** [comment]
@elad335 I see you're the original person that created the fix for the tickets for the graphics issue for the Sonic 2006 game. That's very impressive work. Thank you. Do you know why the patch doesn't work for save states? Sorry I don't know the etiquette for these forums. Thanks again for your patience.

**Comment by MasterChief747** [comment]
Here's my log file while running the save state with bad graphics by the way. Sorry I didn't include one before.
[RPCS3.log](https://github.com/user-attachments/files/16503143/RPCS3.log)

**Comment by digant73** [comment]
is that still an issue?

**Comment by MasterChief747** [comment]
Yes it is an issue. The app crashes when doing save states with this game now.

**Comment by digant73** [comment]
please an update on current build

**Comment by MasterChief747** [comment]
It is still an issue. It says this when you try to make a save state, "The PS3 application has likely crash, you can close it". When you close it it is still running in the background and you have to close it from the Task manager. I hope this can get fixed soon. I've been waiting 2 years to play one of the very hard levels on this game. I forgot to mention the game I am experiencing this issue with is Sonic 2006. Thank you and keep up the good work!
